libnavigation-core/src/main/java/com/mapbox/navigation/core/NavigationVersionSwitchObserver.kt

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.navigation.core
+
+import com.mapbox.navigation.base.options.RoutingTilesOptions
+
+/**
+ *  An interface which enables listening to navigation tiles version switch.
```

Comment:
Let's say more here - explain when and why this happens and recommend what developers should do in case of this event (eg. show an icon that we're in offline mode) to give an example.